### PR TITLE
[IT-1178] Setup SSO access to AWS bridge accounts

### PR DESCRIPTION
**Pull request by zaro0508**
Setup to allow users in the JC bridge admin, developer and iOS developer
groups access to AWS bridge dev and bridge prod accounts.

### CODE REVIEW

**Review by brucehoff — APPROVED** [review]

File: org-formation/700-aws-sso/_tasks.yaml [hunk 1]
```diff
@@ -30,6 +30,153 @@ Parameters:
...
+            "Resource": "arn:aws:s3:::ios-apps.sagebridge.org/*"
+          }
+        ]
+      }
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref MasterAccount
```
Comment (ahayden) [inline org-formation/700-aws-sso/_tasks.yaml:159]: This target should be a bridge acct

**Review by ahayden — CHANGES_REQUESTED** [review]
I think one account target is wrong

File: org-formation/700-aws-sso/_tasks.yaml (hunk 1 above)
Comment (zaro0508) [reply org-formation/700-aws-sso/_tasks.yaml]: fixed by replacing this block with `inlinePolicy` in SsoBridgeProdIosDeveloper

**Review by ahayden — APPROVED** [review]
Looks good

**Pull request merged**